### DISCUSSION TITLE
Low contrast UI

### DISCUSSION THREAD

**Discussion by Flashwalker** [aleksey-hoffman/sigma-file-manager#273] [General]
The UI of the app have really low contrast. It's like you have 50% backlight on your laptop.
How about to make a new theme with higher contrast? (but not as if it were an accessibility mode)

**Comment by aleksey-hoffman** [comment]
Hi, thanks for the feedback
There's a contrast slider in the settings => UI appearance

**Comment by Flashwalker** [comment]
Thanks
